msmbuilder/tests/test_contactfeaturizers.py

```diff
@@ -0,0 +1,85 @@
+import mdtraj as md
+import numpy as np
+import warnings
+
+from msmbuilder.example_datasets import fetch_fs_peptide
+from msmbuilder.featurizer import ContactFeaturizer
+from msmbuilder.featurizer import BinaryContactFeaturizer
+from msmbuilder.featurizer import LogisticContactFeaturizer
+
+def test_contacts():
+    dataset = fetch_fs_peptide()
+    trajectories = dataset["trajectories"]
+    contactfeaturizer = ContactFeaturizer()
+    contacts = contactfeaturizer.transform([trajectories[0]])
+
+    assert contacts[0].shape[1] == 171
+
+
+def test_binaries():
+    dataset = fetch_fs_peptide()
+    trajectories = dataset["trajectories"]
+    binarycontactfeaturizer = BinaryContactFeaturizer()
+    binaries = binarycontactfeaturizer.transform([trajectories[0]])
+
+    assert binaries[0].shape[1] == 171
+    assert sum(sum(binaries[0])) <= binaries[0].shape[0]*binaries[0].shape[1]
```

Comment:
You could just use `np.sum` once, but what might be more appropriate is to check that the `dtype` of this numpy object is `bool`?